sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+        if not entry.content:
+            raise ResourceNotFoundError(
+                "Subscription('Topic: {}, Subscription: {}') does not exist".format(subscription_name, topic_name))
+        subscription = SubscriptionRuntimeInfo._from_internal_entity(entry.content.subscription_description)
+        subscription.name = entry.title
+        return subscription
+
+    async def create_subscription(
+            self, topic: Union[str, TopicDescription], subscription: Union[str, SubscriptionDescription], **kwargs
+    ) -> SubscriptionDescription:
+        """
+
+        :param Union[str, TopicDescription] topic: The topic that will own the to-be-created subscription.
+        :param Union[str, ~azure.servicebus.management.SubscriptionDescription] subscription: The subscription name or a
+        `SubscriptionDescription` instance. When it's a str, it will be the name of the created subscription.
+         Other properties of the created subscription will have default values decided by the ServiceBus.
+        :rtype:  ~azure.servicebus.management.SubscriptionDescription
+        """
+        try:
+            topic_name = topic.name  # type: ignore
+        except AttributeError:
+            topic_name = topic
+        try:
+            subscription_name = subscription.name  # type: ignore
+            to_create = subscription._to_internal_entity()  # type: ignore
+        except AttributeError:
+            subscription_name = subscription  # type: ignore
+            to_create = InternalSubscriptionDescription()  # Use an empty queue description.
+
+        create_entity_body = CreateSubscriptionBody(
+            content=CreateSubscriptionBodyContent(
+                subscription_description=to_create,  # type: ignore
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            entry_ele = cast(
+                ElementTree,
+                await self._impl.subscription.put(
+                    topic_name,
+                    subscription_name,  # type: ignore
+                    request_body, api_version=constants.API_VERSION, **kwargs)
+            )
+
+        entry = SubscriptionDescriptionEntry.deserialize(entry_ele)
+        result = SubscriptionDescription._from_internal_entity(entry.content.subscription_description)
+        result.name = subscription_name
+        return result
+
+    async def update_subscription(
+            self, topic: Union[str, TopicDescription], subscription_description: SubscriptionDescription, **kwargs
+    ) -> None:
+        """Update a subscription.
+
+        Before calling this method, you should use `get_subscription` to get a `SubscriptionDescription` instance,
+        then update the related attributes and call this method.
+        Only a portion of properties can be updated.
+        Refer to TODO: to add the doc link that describes what can be updated.
```

Comment:
LOL, found a `TODO` not done yet.

we should check if there're other "TODOs" in the code and do them :)

Reply:
Removed. I had this TODO because I was waiting for service team to update doc so I can past it here.
It's better to remove them before the link is available.